input/fsh/profiles/RORTask.fsh

```diff
@@ -46,7 +46,7 @@ Description: "Anomalie concerant une donnée dans une ressource du ROR"
 * input[errorValue].value[x] only string
 * input[proposedValue] ^short = "nouvelle valeur proposée"
 * input[proposedValue].type = InputTaskRORCodeSystem#proposedValue
-* input[proposedValue].value[x] only string
+* input[proposedValue].value[x]
```

Comment:
Je pense qu'avant de merge cette PR il faut répondre à ces questions (@jcserafini) :
- Quelles sont les règles de remplissages: quel type est autorisé, dans quelle circonstance ?
- A quoi ressemblera l'interface graphique si c'est un type complexe avec des cardinalités 0..* dedans (comme CodeableConcept) ?
- Est-ce qu'il y aura un date-picker si c'est une date ? Est-ce que ça va jusqu'à l'instant avec sa regex complexe : (https://www.hl7.org/fhir/R4/datatypes.html#instant)
- Dans l'interface graphique, ça sera un string qui sera rentré par l'utilisateur in fine, donc pourquoi proposer cela ?
- Quel type de persona va remplir ce champs ?
- Si le client rentre qqc qui n'est pas compatible FHIR (ex :  la regex instant n'est pas respectée), comment est-ce que c'est géré niveau interface graphique ?

Déterminer cela me semble un prérequis indispensable avant de pousser cette nouvelle version de l'IG.

Reply:
Quelles sont les règles de remplissages: quel type est autorisé, dans quelle circonstance ?
- On veut permettre aux utilisateurs de proposer des valeurs en fonction de l'attribut du modele d'exposition concerné (un codeable concept, un string une date ...)

A quoi ressemblera l'interface graphique si c'est un type complexe avec des cardinalités 0..* dedans (comme CodeableConcept) ?
- Pour chaque type de donnée on va devoir adapter notre interface. Pour le moment notre IHM de déclaration d'anomalie ne permet de saisir que sur une dizaine de champs. On souhaite cependant que le format de la ressource nous permette d'élargir cette liste en fonction de l'usage

Est-ce qu'il y aura un date-picker si c'est une date ? Est-ce que ça va jusqu'à l'instant avec sa regex complexe : (https://www.hl7.org/fhir/R4/datatypes.html#instant)
- Oui, on va permettre de saisir des anos sur les horaires d'ouverture des HS qui sont composés de dateTime. On n'a pas encore développé ça on fera attention aux regex

Dans l'interface graphique, ça sera un string qui sera rentré par l'utilisateur in fine, donc pourquoi proposer cela ?
- justement non, l'utilisateur pourra choisir dans notre IHM la valeur proposée si il estime que celle diffusée est en erreur. A noter qu'on peut aussi se contenter d'une description en string.

Quel type de persona va remplir ce champs ?
- Pour des responsables de systeme numérique consommateur (santé.fr / vt) qui n'ont pas encore interfacé avec Task. Et pour des responsables qualités régionaux. 

Si le client rentre qqc qui n'est pas compatible FHIR (ex : la regex instant n'est pas respectée), comment est-ce que c'est géré niveau interface graphique ?
- on n'a pas encore développé tout ça, Dans notre IHM cela sera impossible de saisir une donnée proposée de format incorrect. Dans un système numérique interfacé , cela pourrait être un souci on va voir ce qu'on affiche dans ce cas. A noter que ce sont des valeurs proposées et non des corrections, il y a toujours action humaine de la part du responsable de la donnée en anomalie.

Dernier point: on souhaiterait que les champs suivants ne soient pas typés en string mais puisse prendre plusieurs types :
[input:errorValue]
[input:proposedValue]
[output:newValue]

Reply:
ok @jcserafini je fais les mêmes modifs pour [input:errorValue]
[input:proposedValue]
[output:newValue]
et je commit :)